docs/spec/src/integration/spec/3-datastructs.md

```diff
@@ -61,7 +61,7 @@ The `blobHeader` is submitted alongside the `blob` as part of the DisperseBlob r
 
 ![image.png](../../assets/integration/blob-certificate.png)
 
-Refer to the eigenda [protobufs](../../protobufs/generated/disperser_v2.md) documentation for full details of this struct.
+Refer to the eigenda [protobufs](https://github.com/Layr-Labs/eigenda/blob/58acdfdae98a897c1eceb4690b7fac4d3e8acb1e/api/proto/disperser/v2/disperser_v2.proto) for full details of this struct.
```

Comment:
I still don't know what the best choice is for links directly to code: specific commit, or `master` 🤔

Reply:
updated to point to master: [4c41a4c](https://github.com/Layr-Labs/eigenda/pull/1637/commits/4c41a4c349445347a54858f942de28439e572db7)

I think the best is prob to point to master so people get the latest... but that only works if we have ways to fix broken links (for example on the docs site you can't merge if link returns a 404). Think mdbook has something similar.. maybe https://github.com/Michael-F-Bryan/mdbook-linkcheck... but don't really feel like setting it up right now :D